src/lang/Messages_zh_CN.properties

```diff
@@ -5,7 +5,7 @@
 # labels that are not really expected to be translated, such as the ports at the end of this file
 
 acsrf.api.action.addOptionToken=\u6dfb\u52a0\u4e00\u4e2a\u53cdCSRF\u4ee4\u724c\u7ed9\u5b9a\u540d\u79f0\uff0c\u9ed8\u8ba4\u542f\u7528
-acsrf.api.action.removeOptionToken=Removes the anti-CSRF token with the given name
+acsrf.api.action.removeOptionToken=Elimina el token CSRF anti con el nombre facilitado
```

Comment:
This is Spanish. I wonder if there's something wrong with Crowdin export?? Following files have also mixed translations.